app/views/posts/_form.html.erb

```diff
@@ -15,7 +15,7 @@
   <%= f.input :body %>
 
   <p>Please select at least two categories.</p>
-  <%= f.association :categories, as: :check_boxes %>
+  <%= f.input :category_ids, as: :check_boxes, collection: Category.all %>
 
```

Comment:
How does simple_form know it has to grab `Category.all` here, @rafaelfranca (@gostosa)? Does that really happen behind the scenes, or does `f.association :categories` only invoke `model.categories` to display the collection items?